test/crd_validation_test.go

```diff
@@ -36,7 +36,6 @@ func TestSampleCustomResources(t *testing.T) {
 	root := "./../deploy/crds"
 	crdCrMap := map[string]string{
 		"ocs.openshift.io_ocsinitializations_crd.yaml": "ocs_v1_ocsinitialization_cr",
-		"ocs.openshift.io_storageclusters_crd.yaml":    "ocs_v1_storagecluster_cr",
```

Comment:
This is not correct. You want to maintain the validation of StorageCluster, you just have to change the validation parameters.